src/main/java/com/websummarizer/Web/Summarizer/controller/WebController.java

```diff
@@ -14,4 +23,23 @@ String getSummary(Model model) {
         return "api/summary"; // webpage to return
     }
 
+    @GetMapping("/")
+    String index(Model model, @AuthenticationPrincipal OAuth2User principal) {
+        Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+
+        // If the user is actively logged in, automatically redirect to pro features site
+        // https://stackoverflow.com/questions/13131122/spring-security-redirect-if-already-logged-in
+        if (!(auth instanceof AnonymousAuthenticationToken)) {
+            //System.out.println(principal.getAttributes());
+            System.out.println(principal.getName());
```

Comment:
Use Logger instead of print statements

Reply:
For now, I have just removed the print statements. I initially kept those because I wanted to keep track of what attributes can be extracted from the Oauth2 authentication request. I just wrote it down in a separate txt file for reference instead.